batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/controller/IndexController.java

```diff
@@ -13,8 +13,8 @@
 @WebServlet({ "/" })
 public class IndexController extends HttpServlet {
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
-        Integer i = TypeCaster.toInt("123");
-        System.out.println("value of i: " + i);
-        req.getServletContext().getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
-    };
+        req.setAttribute("test","testing");
+        resp.sendRedirect("/students");
+//        req.getServletContext().getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)